modules/admin_manual/pages/configuration/user/login_policies.adoc

```diff
@@ -0,0 +1,64 @@
+= Login Policies
+:toc: right
+
+:description: ownCloud provides login policies so admins can allow or reject groups of users to access ownCloud via specific login mechanisms.
```

Comment:
This isn't exactly true. Login policies, in general, will allow or reject users from login. Right now, we only have one login policy implemented which the group login policy, which will allow or reject the user based on the groups and login mechanism (as you've described)